app/Console/Commands/Migrations/CreateMissingAuth0Clients.php

```diff
@@ -89,7 +89,7 @@ private function createMissingAuth0ClientsForIntegration(Integration $integratio
      */
     private function getIntegrationsWithMissingAuth0Clients(): Collection
     {
-        $migratedIntegrationModels = IntegrationModel::query()->has('auth0Clients', '<',  3)->get();
+        $migratedIntegrationModels = IntegrationModel::query()->has('auth0Clients', '<', 3)->get();
```

Comment:
Maybe also use the `count(Auth0Tenant::cases())` here like here? https://github.com/cultuurnet/publiq-platform/pull/163/files#diff-d994cfcee8ed4cd920c17e6dd109c4deac9172d85f18a9788ca72a565e60180bR60

I don't see another one getting added in the foreseeable future, but who knows 🤷‍♂️

Reply:
What did I do there 🙈